api/src/main/java/org/openmrs/module/allergyapi/AllergenType.java

```diff
@@ -14,5 +14,5 @@
 package org.openmrs.module.allergyapi;
 
 public enum AllergenType {
-	DRUG, FOOD, ENVIRONMENT
+	DRUG, FOOD, ENVIRONMENT, OTHER
```

Comment:
I thought you said everything which is not drug or food is other. Do we then need environment?